server/src/main/java/org/elasticsearch/index/query/IntervalsSourceProvider.java

```diff
@@ -548,6 +548,18 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         public static Prefix fromXContent(XContentParser parser) throws IOException {
             return PARSER.parse(parser, null);
         }
+
+        public String getPrefix() {
```

Comment:
nit: maybe we can keep this method and the other two getters package private for now, since its only used in the test.